src/renderer/components/wallet/txs/send/SendFormETH.tsx

```diff
@@ -0,0 +1,341 @@
+              O.getOrElse(() => '...')
+            ),
+          (error) => `${intl.formatMessage({ id: 'common.error' })} ${error?.message ?? ''}`,
+          (fees) =>
+            formatAssetAmountCurrency({
+              amount: baseToAsset(fees[selectedFeeOptionKey]),
+              asset: AssetETH,
+              trimZeros: true
+            })
+        )
+      ),
+    [feesRD, intl, selectedFeeOptionKey]
+  )
+
+  const renderFeeError = useMemo(() => {
+    if (!isFeeError) return <></>
+
+    const amount = FP.pipe(
+      oEthAmount,
+      // no eth asset == zero amount
+      O.getOrElse(() => ZERO_ASSET_AMOUNT)
+    )
+
+    const msg = intl.formatMessage(
+      { id: 'wallet.errors.fee.notCovered' },
+      {
+        balance: formatAssetAmountCurrency({
+          amount,
+          asset: AssetETH,
+          trimZeros: true
+        })
+      }
+    )
+
+    return (
+      <Styled.Label size="big" color="error">
+        {msg}
+      </Styled.Label>
+    )
+  }, [oEthAmount, intl, isFeeError])
+
+  const feeOptionsLabel: Record<FeeOptionKey, string> = useMemo(
+    () => ({
+      fast: intl.formatMessage({ id: 'wallet.send.fast' }),
+      fastest: intl.formatMessage({ id: 'wallet.send.fastest' }),
+      average: intl.formatMessage({ id: 'wallet.send.average' })
+    }),
+    [intl]
+  )
+
+  const renderFeeOptionsRadioGroup = useCallback(() => {
+    const onChangeHandler = (e: RadioChangeEvent) => setSelectedFeeOptionKey(e.target.value)
+
+    return (
+      <StyledR.Radio.Group onChange={onChangeHandler} value={selectedFeeOptionKey} disabled={isLoading}>
+        <StyledR.Radio value="fastest" key="fastest">
+          <StyledR.RadioLabel>{feeOptionsLabel['fastest']}</StyledR.RadioLabel>
+        </StyledR.Radio>
+        <StyledR.Radio value="fast" key="fast">
+          <StyledR.RadioLabel>{feeOptionsLabel['fast']}</StyledR.RadioLabel>
+        </StyledR.Radio>
+        <StyledR.Radio value="average" key="average">
+          <StyledR.RadioLabel>{feeOptionsLabel['average']}</StyledR.RadioLabel>
+        </StyledR.Radio>
+      </StyledR.Radio.Group>
+    )
+  }, [feeOptionsLabel, isLoading, selectedFeeOptionKey])
+
+  const renderFeeOptions = useMemo(
+    () =>
+      FP.pipe(
+        oFees,
+        O.fold(
+          () =>
+            // render radio group while reloading fees
+            FP.pipe(
+              prevFeesRef.current,
+              O.map(renderFeeOptionsRadioGroup),
+              O.getOrElse(() => <></>)
+            ),
+          renderFeeOptionsRadioGroup
+        )
+      ),
+    [prevFeesRef, oFees, renderFeeOptionsRadioGroup]
+  )
+
+  const addressValidator = useCallback(
+    async (_: unknown, value: string) => {
+      if (!value) {
+        return Promise.reject(intl.formatMessage({ id: 'wallet.errors.address.empty' }))
+      }
+      if (!addressValidation(value.toLowerCase())) {
+        return Promise.reject(intl.formatMessage({ id: 'wallet.errors.address.invalid' }))
+      }
```

Comment:
To validate ETH address use [`utils.validateAddress()`](https://github.com/xchainjs/xchainjs-lib/blob/525acb1213982a18c8438dcdd9a1d5b840b915cd/packages/xchain-ethereum/src/utils.ts#L52-L59) of `xchain-ethereum`